src/classes/UTIL_JobProgress_CTRL.cls

```diff
@@ -87,10 +87,14 @@ public with sharing class UTIL_JobProgress_CTRL {
         }
 
         Integer completedCount = 0;
-        //Query the Batch apex jobs
+        //Query the Batch apex jobs, excluding skew dispatcher jobs
+        String skewDispatcherClassName =  CRLP_SkewDispatcher_BATCH.class.getName();
```

Comment:
Like with the prevent dupes code, this will need to strip out the namespace from the class name before using in the query.